src/main/java/world/inetum/realdolmen/inetumrealJobs/entities/Account.java

```diff
@@ -0,0 +1,141 @@
+package world.inetum.realdolmen.inetumrealJobs.entities;
+
+import world.inetum.realdolmen.inetumrealJobs.entities.enums.Gender;
+
+import javax.persistence.*;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Past;
+import java.time.LocalDate;
+
+@Entity
+@Table(name = "Account")
+@Inheritance(strategy = InheritanceType.JOINED)
+public class Account extends BaseModel {
```

Comment:
Should be abstract I guess, as it's either a job seeker or a recruiter and not just an account?